src/pages/home/report/ReportActionCompose.js

```diff
@@ -953,7 +953,7 @@ class ReportActionCompose extends React.Component {
                                                     style={[
                                                         styles.dFlex,
                                                         styles.flexColumn,
-                                                        isFullComposerAvailable || this.props.isComposerFullSize ? styles.justifyContentBetween : styles.justifyContentEnd,
+                                                        isFullComposerAvailable || this.props.isComposerFullSize ? styles.justifyContentBetween : styles.justifyContentCenter,
```

Comment:
`Add action` (`+`) icon is moving up and down due to this change. Let's revert it back to `justifyContentEnd`. And change height of style.composerSizeButton to 40 if it makes sense and solves the issue you're trying to solve by this change.

Reply:
It looks bad than before. There is a huge change in height on adding 3rd line but no change at all on 4th.
https://github.com/Expensify/App/assets/25876548/9f56367a-5515-45c9-8bc9-bf9c8f3edcaa

Reply:
@daraksha-dk Thanks for testing different scenarios, let's revert back to `justifyContentCenter `